pom.xml

```diff
@@ -211,7 +212,7 @@
     <soa.model.core.version>1.2.2</soa.model.core.version>
     <httpclient.version>4.2.3</httpclient.version>
     <spring-modules-ojb.version>0.8a</spring-modules-ojb.version>
-    <spring.version>3.2.3.RELEASE</spring.version>
+    <spring.version>3.2.18.RELEASE</spring.version>
```

Comment:
This updates to the newest version of Spring 3 and things seem to generally work with it.  We could consider also upgrading to Spring 4 by backporting some changes from KualiCo's version of Rice, but I'm thinking that should be a separate PR if we decide to pursue that.